scripts/ubuntu.sh

```diff
@@ -0,0 +1,222 @@
+apt_package_list="software-properties-common \
+                  python3-pip \
+                  wget \
+                  build-essential \
+                  git \
+                  cmake \
+                  gfortran \
+                  libblas-dev \
+                  liblapack-dev \
+                  libeigen3-dev \
+                  libhdf5-dev \
+                  hdf5-tools"
+
+# list of python package required for PyNE and its depedencies (installed using pip3 python package manager)
+pip_package_list="numpy==1.23 \
+                  scipy \
+                  cython \
+                  nose \
+                  tables \
+                  matplotlib \
+                  jinja2 \
+                  setuptools \
+                  future \
+                  progress"
+
+# function to check if build folder already exists
+function check_repo() {
+
+    repo_name=$1
+
+    if [ -d ${repo_name} ] ; then
+        read -p "Delete the existing ${repo_name} directory and all contents? (y/n) " -n 1 -r
+        if [[ $REPLY =~ ^[Yy]$ ]] ; then
+            rm -rf ${repo_name}
+        fi
+    fi
+
+}
+
+# system update
+sudo apt-get -y update
+sudo apt-get install -y --fix-missing ${apt_package_list}
+apt-get clean -y;
+update-alternatives --install /usr/bin/python python /usr/bin/python3 10;
+update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 10;
+pip install --user --upgrade pip;
+pip install --user ${pip_package_list}
+
+install_dir=${HOME}/opt
+mkdir -p ${install_dir}
+
+# hdf5 std directory
+hdf5_libdir=$HOME/opt/hdf5/$build_hdf5
+
+# need to put libhdf5.so on LD_LIBRARY_PATH (Making sure that LD_LIBRARY_PATH is defined first)
+if [ -z $LD_LIBRARY_PATH ]; then
+  export LD_LIBRARY_PATH="${hdf5_libdir}"
+else
+  export LD_LIBRARY_PATH="${hdf5_libdir}:$LD_LIBRARY_PATH"
+fi
+
+
+############
+### MOAB ###
+############
+# pre-setup
+cd ${install_dir}
+check_repo moab
+mkdir -p moab
+cd moab
+
+# clone and version
+git clone --depth 1 --single-branch -b 5.3.0 https://bitbucket.org/fathomteam/moab moab-repo
+cd moab-repo
+mkdir -p build
+cd build
+
+# cmake, build and install
+cmake ../ -DENABLE_PYMOAB=ON \
+          -DCMAKE_INSTALL_PREFIX=${install_dir}/moab \
+          -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_libdir} \
+          -DBUILD_SHARED_LIBS=ON \
+          -DENABLE_BLASLAPACK=OFF \
+          -DENABLE_FORTRAN=OFF    
+make -j 3 
+cd pymoab
+pip install --user .
+cd ..
+make install
+cd ..
+rm -rf moab-repo
+
+# Adding MOAB/lib to $LD_LIBRARY_PATH and $LIBRARY_PATH
+export LD_LIBRARY_PATH="${install_dir}/moab/lib:$LD_LIBRARY_PATH"
+
+# Adding pymoab to $PYTHONPATH
+PYTHON_VERSION=$(python -c 'import sys; print(sys.version.split('')[0][0:3])')
+if [ -z $PYTHONPATH ]; then
+  export PYTHONPATH="$install_dir/moab/lib/python${PYTHON_VERSION}/site-packages"
+else
+  export PYTHONPATH="$install_dir/moab/lib/python${PYTHON_VERSION}/site-packages:$PYTHONPATH"
+fi
```

Comment:
Might not need this if we install with `pip` - see how this works in the dockerfile?